apps/tehwolfde/src/assets/styles/_tokens.scss

```
@@ -0,0 +1,86 @@
+  --tw-neutral-050: #f6f6f6;
+
+  --tw-glass-blur: blur(50px);
+}
+
+// Dark — the original palette, unchanged from what shipped.
+body.dark {
+  --tw-bg: var(--tw-neutral-900);
+  --tw-surface: var(--tw-neutral-850);
+  --tw-surface-raised: var(--tw-neutral-800);
+  --tw-control-bg: var(--tw-neutral-700);
+  // The accent is only 3.79:1 on the control surface, so controls get their
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)